package/kedro_viz/integrations/kedro/sqlite_store.py

```diff
@@ -42,10 +43,22 @@ def to_json(self) -> str:
         """Returns session_store information in json format after converting PosixPath to string"""
         session_dict = {}
         for key, value in self.data.items():
-            if _is_json_serializable(value):
+            # Get branch info through git_sha
+            if key == "git":
+                sha = [v for k, v in value.items() if k == "commit_sha"]
+                git = subprocess.run(
+                    ("git", "name-rev", "--name-only", sha[0]),
+                    stdout=subprocess.PIPE,
+                    check=True,
+                )
+                branch = git.stdout.decode("utf-8")
+                value["branch"] = branch
                 session_dict[key] = value
```

Comment:
Is the value here always `_is_json_serializable` or should it still go through that check?